[mealie-next] - Dev Container is Broken

### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

Attempting to open the project in the dev container (as described [in the docs](https://nightly.mealie.io/contributors/developers-guide/starting-dev-server/#with-vscode-dev-containers)) fails with the following exception:

<details>
<summary>
Console Output
</summary>

```
  • Installing six (1.16.0): Failed

  CalledProcessError

  Command '['/usr/local/bin/python3.10', '-m', 'pip', 'install', '--disable-pip-version-check', '--isolated', '--no-input', '--prefix', '/usr/local', '--no-deps', '/home/vscode/.cache/pypoetry/artifacts/89/b2/f8/fd92b6d5daa0f8889429b2fc67ec21eedc5cae5d531ee2853828ced6c7/six-1.16.0-py2.py3-none-any.whl']' returned non-zero exit status 1.

  at /usr/local/lib/python3.10/subprocess.py:526 in run
       522│             # We don't call process.wait() as .__exit__ does that for us.
       523│             raise
       524│         retcode = process.poll()
       525│         if check and retcode:
    →  526│             raise CalledProcessError(retcode, process.args,
       527│                                      output=stdout, stderr=stderr)
       528│     return CompletedProcess(process.args, retcode, stdout, stderr)
       529│ 
       530│ 

The following error occurred when trying to handle this error:


  EnvCommandError

  Command ['/usr/local/bin/python3.10', '-m', 'pip', 'install', '--disable-pip-version-check', '--isolated', '--no-input', '--prefix', '/usr/local', '--no-deps', '/home/vscode/.cache/pypoetry/artifacts/89/b2/f8/fd92b6d5daa0f8889429b2fc67ec21eedc5cae5d531ee2853828ced6c7/six-1.16.0-py2.py3-none-any.whl'] errored with the following return code 1, and output: 
  Processing /home/vscode/.cache/pypoetry/artifacts/89/b2/f8/fd92b6d5daa0f8889429b2fc67ec21eedc5cae5d531ee2853828ced6c7/six-1.16.0-py2.py3-none-any.whl
  Installing collected packages: six
  ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/usr/local/lib/python3.10/site-packages/six.py'
  Consider using the `--user` option or check the permissions.
  
  

  at /opt/poetry/venv/lib/python3.10/site-packages/poetry/utils/env.py:1540 in _run
      1536│                 output = subprocess.check_output(
      1537│                     command, stderr=subprocess.STDOUT, env=env, **kwargs
      1538│                 )
      1539│         except CalledProcessError as e:
    → 1540│             raise EnvCommandError(e, input=input_)
      1541│ 
      1542│         return decode(output)
      1543│ 
      1544│     def execute(self, bin: str, *args: str, **kwargs: Any) -> int:

The following error occurred when trying to handle this error:


  PoetryException

  Failed to install /home/vscode/.cache/pypoetry/artifacts/89/b2/f8/fd92b6d5daa0f8889429b2fc67ec21eedc5cae5d531ee2853828ced6c7/six-1.16.0-py2.py3-none-any.whl

  at /opt/poetry/venv/lib/python3.10/site-packages/poetry/utils/pip.py:58 in pip_install
       54│ 
       55│     try:
       56│         return environment.run_pip(*args)
       57│     except EnvCommandError as e:
    →  58│         raise PoetryException(f"Failed to install {path.as_posix()}") from e
       59│ 

make: *** [makefile:39: setup] Error 1
[27607 ms] onCreateCommand failed with exit code 2. Skipping any further user-provided commands.
[27608 ms] Error: Command failed: /bin/sh -c sudo chown -R vscode:vscode /workspaces/mealie/frontend/node_modules && make setup
[27608 ms]     at HL (/root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js:1691:137)
[27608 ms]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[27608 ms]     at async Promise.all (index 0)
[27609 ms]     at async Jg (/root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js:1683:3580)
[27609 ms]     at async Wg (/root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js:1683:2644)
[27609 ms]     at async mue (/root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js:2020:27351)
[27609 ms]     at async pue (/root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js:2020:25040)
[27645 ms] Exit code 1
[27646 ms] Start: Run: docker rm -f f1e389db89921d01b3afc763b899e097acac749980926a2ef6ee282284237f7d
[27654 ms] Command failed: node /root/.vscode-remote-containers/dist/dev-containers-cli-0.266.1/dist/spec-node/devContainersSpecCLI.js run-user-commands --workspace-folder /workspaces/mealie --id-label vsch.local.repository=https://github.com/hay-kot/mealie.git --id-label vsch.local.repository.volume=mealie-16d7abf94e700cb234b893c6829cd704 --id-label vsch.local.repository.folder=mealie --container-id 998c55d17b056f045612fa00d7cbdd81a327c194dc3a7f7154dae43ed3205376 --log-level debug --log-format json --config /workspaces/mealie/.devcontainer/devcontainer.json --override-config /tmp/devcontainer-207a5dd9-bf42-42c5-949e-a8f7dd0da3d3.json --default-user-env-probe loginInteractiveShell --skip-non-blocking-commands true --prebuild false --stop-for-personalization true --remote-env SSH_AUTH_SOCK=/tmp/vscode-ssh-auth-57d0d86a742edd1ade41adb8ad7a26d7c14ba73e.sock --remote-env REMOTE_CONTAINERS_IPC=/tmp/vscode-remote-containers-ipc-57d0d86a742edd1ade41adb8ad7a26d7c14ba73e.sock --remote-env REMOTE_CONTAINERS=true --mount-workspace-git-root true
[27654 ms] Exit code 1
```

</details>

### Deployment

Other

### Deployment Details

You can reproduce this with a fresh clone, e.g. using the `Remote: Clone Repository in Container Volume` action in VS Code.

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -20,8 +20,7 @@ ENV PYTHONUNBUFFERED=1 \
     PIP_NO_CACHE_DIR=off \
     PIP_DISABLE_PIP_VERSION_CHECK=on \
     PIP_DEFAULT_TIMEOUT=100 \
-    POETRY_HOME="/opt/poetry" \
-    POETRY_VIRTUALENVS_CREATE=false
+    POETRY_HOME="/opt/poetry"
 
 # prepend poetry and venv to path
 ENV PATH="$POETRY_HOME/bin:$PATH"

